content/docs/reference/cli/environment-variables.md

```diff
@@ -124,6 +124,18 @@ meta_desc: A list of different environment variables the Pulumi CLI supports.
         </p>
         <pre><code class="text-xs">PULUMI_ACCESS_TOKEN="your-access-token"</code></pre>
     </dd>
+    <dt>
+        <span class="font-mono">
+            PULUMI_BACKEND_URL
+        </span>
+    </dt>
+    <dd>
+        <p>
+            Set this environment variable to use a sepcified backend instead of the default "current" backend from the project
+            file or from a `pulumi login`.  See [State and Backends]({{< relref "/docs/intro/concepts/state" >}}) for details on valid backend URLs.
```

Comment:
```suggestion
            Set this environment variable to use a specified backend instead of the current backend for the project.  See [State and Backends]({{< relref "/docs/intro/concepts/state" >}}) for details on valid backend URLs.
```